packages/cypress/internal/run-cypress.js

```diff
@@ -24,9 +24,9 @@ async function main() {
     // Detect that we are running as a test, by using well-known environment
     // variables. See go/test-encyclopedia
     if (!process.env.BUILD_WORKSPACE_DIRECTORY) {
-      await invokeCypressWithCommand('run');
+      await invokeCypressWithCommand('run-ct');
     } else {
-      await invokeCypressWithCommand('open');
+      await invokeCypressWithCommand('open-ct');
```

Comment:
Temporary changes, we would need to expose these different commands somehow, perhaps with a rule `cypress_ct_test`?

Reply:
Yeah, I think your intuition to produce a new rule is a good one. That's probably the missing piece of the puzzle